src/pybind/mgr/dashboard_v2/services/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+
+class Service(object):
+    """
+    Base class for all services.
+    """
+    pass
```

Comment:
IMO, as a developer, I'm in favor of having an explicit way to see which properties are injected automatically when extending some class.

Is is possible to add a field to this class that "documents" the injected field (`mgr = None`)?

Maybe this will also help the IDEs.